memdx/client.go

```diff
@@ -2,11 +2,14 @@ package memdx
 
 import (
 	"errors"
+	"os"
 	"sync"
 
 	"go.uber.org/zap"
 )
 
+var enablePacketLogging bool = os.Getenv("GCBCX_PACKET_LOGGING") != ""
```

Comment:
Does this mean that it can't be dynamically toggled?

Reply:
At the moment, you could only turn it on/off globally at application startup.